css/main.css

```diff
@@ -4974,3 +4974,4 @@ body.tg-comingsoonpage .tg-foorterbar .tg-socialicons{padding:0;}
   color: white;
 }
 
+.tg-description{ display: none}
```

Comment:
Just leave a blank line at the end of the file.

Reply:
Sure. Will do.

Reply:
Once you do that, just ping me, and I'll merge the PR.

Reply:
Done.